apps/web/src/components/forms/signin.tsx

```diff
@@ -369,7 +371,7 @@ export const SignInForm = ({
               onClick={onSignInWithOIDCClick}
             >
               <FaIdCardClip className="mr-2 h-5 w-5" />
-              OIDC
+              {oidcProviderLabel}
```

Comment:
Suggest we do `{oidcProviderLabel || 'OIDC'}` since the prop can be undefined and people on older versions of Documenso may not add this to the env upon upgrading.

Reply:
@Mythie  thanks, I have update the code.